common/lib/xmodule/xmodule/tests/test_sequence.py

```diff
@@ -97,6 +97,7 @@ def test_student_view_init(self):
     def test_render_student_view(self):
         html = self._get_rendered_student_view(
             self.sequence_3_1,
+            requested_child='first',
```

Comment:
Hmm.. I **do** think it invalidates the test.  As this test wants to ensure that the default position is the "first" position when a child is not explicitly requested.

The position is stored in the Courseware StudentModule SQL table - which should be getting cleared between each test run.  So I'm surprised that it's related to testing order.  

Unless, there's a cache for that data and it's not being cleared between test runs.  You can test that theory by adding the `CacheIsolationMixin` to this Test class and setting the following class variable:
```
ENABLED_CACHES = ['default', 'mongo_metadata_inheritance', 'loc_cache']
```

(Has this test ever been flaky on Jenkins since this test was introduced?)